docker/README.md

```diff
@@ -251,8 +249,33 @@ Follow the instructions below to set up and run the Harpin AI Toolkit in a Docke
    Enter selection [1-2]: 1
    
    Saving source configuration to .harpin/sources/test.yml...
+
+   Select an option:
+
+        1. Add source
+        2. Update source
+        3. Delete source
+        4. Show sources
+        5. Back to main menu
+
+   Enter selection [1-5]: 5
+   
+   Select an option:
+
+        1. Manage sources
+        2. Run identity resolution
+        3. Export config to S3
+        4. Exit
+
    ```
 
+   The toolkit also supports accessing input files in s3 by specifying the s3 URI (e.g. s3://<bucket_name>/<key>).  If credentials are needed to access files in s3, the following environments are supported and should be set prior to starting the harpin AI toolkit containers.
```

Comment:
Should we replace "the following environments are supported" with "the following environment variables are supported"?